src/RelayProvider.ts

```diff
@@ -0,0 +1,255 @@
+import type { Transaction } from 'ethers';
+import type { Networkish } from "@ethersproject/networks";
+import { JsonRpcProvider, TransactionReceipt } from "@ethersproject/providers";
+import type { ConnectionInfo } from "@ethersproject/web";
+import { getEnvelopingConfig } from './utils';
+import RelayClient, { RequestConfig } from './RelayClient';
+import log from 'loglevel';
+import { RelayHub__factory } from '@rsksmart/rif-relay-contracts';
+import AccountManager from './AccountManager';
+import type { UserDefinedEnvelopingRequest } from './common/relayRequest.types';
+import { useEnveloping } from './utils';
+import BigNumber from 'bignumber.js';
```

Comment:
Remove this. Since The BigNumber library needs to use here is BigNumber from ethers.

Reply:
got it